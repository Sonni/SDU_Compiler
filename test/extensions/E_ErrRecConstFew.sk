var rec: record of {a: int, b: bool};

allocate rec;

rec = {42};

write rec.a;
write rec.b;
