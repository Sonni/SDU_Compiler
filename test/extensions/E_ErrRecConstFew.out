Type checker should output an error since all the values of a record must be
assigned a value when using this feature.